js/reducers/dashboard-reducer.js

```diff
@@ -13,6 +13,8 @@ export default function dashboardReducer (state = INITIAL_DASHBOARD_STATE, actio
       return state.setIn(['expandedActivities', action.activityId.toString()], action.value)
     case SET_STUDENT_EXPANDED:
       return state.setIn(['expandedStudents', action.studentId.toString()], action.value)
+    case SET_STUDENT_SORT:
+      return state.setIn(['sortBy'], action.value)
```

Comment:
I guess `setIn` isn't necessary here, `.set('sortBy', action.value)` would work just fine, but it's pretty much whatever.